bucket/dopamine.json

```diff
@@ -11,6 +11,7 @@
             "Dopamine"
         ]
     ],
+    "persist": "Dopamine\\Settings.xml",
```

Comment:
On a fresh installation, does this file already exist? Or is it created later?

Reply:
Could be both. I don't know cause I haven't intalled it yet. If the file is created later, can scoop create that file? Of course, the file content will be edited by the app itself, or in worse scenario the file will be tampered with the new settings applied.
The first time I know scoop can persist _file_ (not just folder) is after installing Jetbrains IDEs via scoop. It also the first time I installed those IDEs.

Reply:
If the file doesn't exist on a fresh installation, Scoop will assume the item is a _folder_ and create a folder hard link. To create the file, you can do something like this: https://github.com/ScoopInstaller/Extras/blob/943b50ec4541a630533d695a25edf1f54e4d65f3/bucket/q-dir.json#L39

Reply:
What is that `start.qdr`?